docs/guides/developer/docs/architecture/api/types.md

```diff
@@ -482,4 +482,28 @@ Type          | Description
 `boolean`     | `true` or `false`
 `enumeration` | One of the values provided by the field `values`
 
+## Playlists
 
+### Playlist
+
+A playlist is represented as a JSON object or XML with the following fields:
+
+| Field                  | Type                                                     | Description                                                                 |
+|------------------------|----------------------------------------------------------|-----------------------------------------------------------------------------|
+| `id`                   | [`long`](types.md#basic)                                 | Identifier. Will be created if not set.                                     |
+| `entries`              | [`List<PlaylistEntry>`](types.md#PlaylistEntry)          | List of entries.                                                            |
+| `title`                | [`string`](types.md#basic)                               | Metadata.                                                                   |
+| `description`          | [`string`](types.md#basic)                               | Metadata.                                                                   |
+| `creator`              | [`string`](types.md#basic)                               | Metadata.                                                                   |
+| `updated`              | [`Date`](types.md#Date)                                  | Metadata. Updated by Opencast with current Date when playlist gets updated. |
+| `accessControlEntries` | [`List<AccessControlEntry`](types.md#AccessControlEntry) | List of access control entries, basically an ACL.                           |
```

Comment:
Would `acl` not be a shorter and equally fitting name compared to `accessControlEntries`?

Reply:
The result of letting a programmer naming things, for whom there is a real difference between `List<AccessControlEntry` and `AccessControlList`. Don't mind changing that if you want it to be changed.

Reply:
:shrug: if you say thats a meaningful difference I believe you. I don't know the details

Reply:
Ah, what I meant to say was that it is meaningful to someone looking at Opencast code (for too long). Not necessarily to someone getting data from an endpoint.